pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: qr
-version: 3.0.0
+version: 3.1.0
```

Comment:
just 3.0.0 – we haven't published v3 yet

Reply:
Done!